update.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eo pipefail
+
+cd "$(dirname "$BASH_SOURCE")"
+
+versions=( "$@" )
+if [ ${#versions[@]} -eq 0 ]; then
+	versions=( */ )
+fi
+versions=( "${versions[@]%/}" )
+
+for version in "${versions[@]}"; do
+	md5="$(curl -fsSL --retry 5 "https://sourceforge.net/projects/geonetwork/files/GeoNetwork_opensource/v${version}/geonetwork.war.MD5/download")"
+
+	sed -ri \
+		-e 's/^(ENV GN_VERSION) .*/\1 '"$version"'/' \
+		-e 's/^(ENV GN_DOWNLOAD_MD5) .*/\1 '"$md5"'/' \
+		-e 's/^(FROM geonetwork):.*/\1:'"$version"'/' \
+		"$version/Dockerfile" "$version/postgres/Dockerfile"
+done
```

Comment:
Sure, that's fair, but from the perspective of the official images, we definitely won't know the WAR file changed, and certainly won't auto-rebuild unless there's something more active to bust the cache and trigger a rebuild (hence the manual MD5 update).  This also helps add a little bit of process to ensure that the WAR file update is intentional and not malicious, for example.

On many of the repos we maintain, we have an automated process which runs our `./update.sh` scripts, and I'd be happy to discuss adding this repo to our existing tooling. :+1: